src/components/UserProfile/Badges.jsx

```diff
@@ -75,7 +74,6 @@ const Badges = (props) => {
                       allBadgeData={props.allBadgeData}
                       userProfile={props.userProfile}
                       setUserProfile={props.setUserProfile}
-                      isAdmin={props.isAdmin}
```

Comment:
Do we need to change this line to `role={props.role}` as line 58?

Reply:
So far there is no need to pass the requestor role as props to that component